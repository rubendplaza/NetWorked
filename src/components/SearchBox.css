input {
    color: #FFFFFF;
    /* border-color: #66FCF1; */
    border-color: #45A29E;
    background-color: #1F2833;
}

input:hover {
    background-color: #1F2833;
}

input::placeholder {
    color: #FFFFFF;
    opacity: 1;
}

input:focus {
    outline: none;
}

.search {
    display: flex;
    align-items: center;
    justify-content: center;
}