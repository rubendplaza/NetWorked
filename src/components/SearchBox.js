import React from 'react';
import './SearchBox.css';

const SearchBox = ({ searchChange }) => {
    return (
        <div className='ma4 search'>
            <input
            className='bw2 pa3 ba' 
            type='search' 
            placeholder='Search Network' 
            onChange={searchChange}
            />
        </div>
    );
}

export default SearchBox;