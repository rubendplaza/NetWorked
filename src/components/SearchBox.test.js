import React from "react";
import SearchBox from "./SearchBox";
import { shallow } from "enzyme";

test("SearchBox component renders correctly without crashing", () => {
  expect(shallow(<SearchBox />)).toMatchSnapshot();
});
