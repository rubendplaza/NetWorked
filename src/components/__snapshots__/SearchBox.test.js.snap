// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBox component renders correctly without crashing 1`] = `
<div
  className="ma4 search"
>
  <input
    aria-label="Search Network Connections"
    className="bw2 pa3 ba"
    id="search"
    placeholder="Search Network"
    type="search"
  />
</div>
`;
